lib/glueby/internal/wallet/active_record/wallet.rb

```diff
@@ -12,11 +12,14 @@ class Wallet < ::ActiveRecord::Base
           validates :wallet_id, uniqueness: { case_sensitive: false }
 
           # @param [Tapyrus::Tx] tx
-          def sign(tx)
+          # @param [Array] prevtxs array of outputs
+          def sign(tx, prevtxs = [])
             tx.inputs.each.with_index do |input, index|
```

Comment:
There doesn't seem to be a test case in `wallet_spec.rb` that specifies `prevtxs`.

Reply:
Added test case for AR::Wallet#sign, (932eca58775c6a2b6f6b91dfb7b7f712132c1020)